python/eeg2bids.py

```diff
@@ -40,29 +50,49 @@ def tarfile_bids(sid, data):
     print('response received!')
     print(response)
     send = {
-            'compression_time': response['compression_time']
-        }
+        'compression_time': response['compression_time']
+    }
     print('send received!')
     print(send)
     sio.emit('response', send)
 
 
+@sio.event
+def set_loris_credentials(sid, data):
+    print('set_loris_credentials:', data)
+    lorisCredentials = data
+    loris_api.url = lorisCredentials.lorisURL
+    loris_api.username = lorisCredentials.lorisUsername
+    loris_api.password = lorisCredentials.lorisPassword
+    loris_api.login()
+    sio.emit('loris_sites', loris_api.get_sites())
```

Comment:
Is there s reason why this is called here?

Reply:
@laemtl yes, since it makes most sense to emit your functions after loris_api.login() and so the user interface gets the updated LORIS data right after it. Notice in the javascript that I commented the emits to those functions.

Reply:
Makes perfect sense, thanks!